Makefile

```diff
@@ -78,7 +78,7 @@ deploy: manifests deploy-namespace-rbac deploy-manager ## Deploy operator in the
 
 deploy-dev: docker-build-dev patch-dev manifests deploy-namespace-rbac docker-registry-secret deploy-manager-dev ## Deploy operator in the configured Kubernetes cluster in ~/.kube/config, with local changes
```

Comment:
I'd consider adding `check-env-docker-repo` to the start of the `deploy-dev` target so it runs before everything else; it's strictly already a prerequisite of this target in `docker-registry-secret`, but given running `make deploy-dev` is such a common operation in the team, I think it's worth it to fail fast in this case where one forgets to set the env vars. Up to you though!

Reply:
Done. I like the idea.